content/1_General/Input_Output.mdx

```diff
@@ -552,20 +579,23 @@ public class Main {
 
 ```java
 import java.io.*;
-import java.util.StringTokenizer;
 
 public class Main {
 	static BufferedReader r = new BufferedReader(new InputStreamReader(System.in));
 	static PrintWriter pw = new PrintWriter(System.out);
```

Comment:
maybe move these into the main method as well (no reason for them to be static)

Reply:
yeah i thought that too, just didn't wanna change the code too much

Reply:
done